### ISSUE TITLE
Transparent Background?

### ISSUE THREAD

**Issue by PeterTheSalmon** [bow-swift/nef-plugin#24]
Is there any way to remove the background and simply have the code snippet on its own? Similar to this image:
![Nef](https://user-images.githubusercontent.com/87033324/142358865-cae822c2-8644-4e22-a939-7510131111c4.png)

Thanks!

**Comment by miguelangel-dev** [comment]
Yep, it would be a nice improvement. But I do not have enough time for addressing it right now. I have just created [the next issue](https://github.com/bow-swift/nef/issues/183) in `nef` repository.
